src/test/java/com/hillel/items_exchange/security/SecurityConfigIntegrationTest.java

```diff
@@ -88,13 +87,13 @@ public void loginWithValidUserIsOk() throws Exception {
     @Test
     @Transactional
     @DataSet("database_init.yml")
-    public void loginWithNotValidUserIsOk() throws Exception {
+    void loginWithNotValidUserIsOk() throws Exception {
```

Comment:
Name of this test does not reflect its functionality

Reply:
I'll update it.